packages/core/src/ObjectContainer.js

```diff
@@ -484,14 +484,28 @@ export default class ObjectContainer {
    *         implementation is known to this object container.
    */
   _getEntry(name) {
+    let nameToGet = name;
+
+    if (Array.isArray(name)) {
+      nameToGet = name?.[0];
```

Comment:
When it's array the first argument is always entryname, so you can remove the optional chaining, but see below.